packages/tools/webpack-fluid-loader/src/loader.ts

```diff
@@ -279,9 +286,13 @@ export async function start(
         if (window.location.hash === "#prefetch") {
             assert(options.mode === "spo-df" || options.mode === "spo",
                 0x1ea /* "Prefetch snapshot only available for odsp!" */);
+
+            const resolvedUrl = await urlResolver.resolve({ url: documentUrl });
+            assert(resolvedUrl !== undefined, "resolvedUrl is undefined");
+
             const prefetched = await prefetchLatestSnapshot(
-                await urlResolver.resolve({ url: documentUrl }),
-                async () => options.odspAccessToken,
+                resolvedUrl,
+                async () => options.odspAccessToken ?? null,
```

Comment:
I think are coalescing to `null` here because of `isTokenFromCache` not handling `undefined` correctly?  If so, maybe add a comment about why we're using `null` instead of the preferred `undefined`.